pkg/providers/instancetypes/instancetypes.go

```diff
@@ -195,7 +195,7 @@ func (p *InstanceTypeProvider) getInstanceTypeZones(ctx context.Context, nodeTem
 }
 
 // getInstanceTypes retrieves all instance types from the ec2 DescribeInstanceTypes API using some opinionated filters
-func (p *InstanceTypeProvider) getInstanceTypes(ctx context.Context) ([]*ec2.InstanceTypeInfo, error) {
+func (p *Provider) GetInstanceTypes(ctx context.Context) ([]*ec2.InstanceTypeInfo, error) {
```

Comment:
Is there a reason that this `getInstanceTypes` call needs to be publicly exposed?

Reply:
This function is being tested directly. I didn't see a way to test it without making it public.